frontend/components/About.jsx

```diff
@@ -0,0 +1,73 @@
r3a1VdLEdFaYZ8qItYR7FHrNVm6mvbA7IdVdxSxgGZzyIGUdCdZ0Vum3tFbnMZun7GwLKOHfiasCA4tLhIY1gOZ8czyHXRGErzhKbs0zTrPmIkfnDMeYVf7RKrMNhqLaVQubVY4MplwgtBbUNSYvMsF55QrDB4UswtKmfhwzWkDTjyz969X4mHTNPF+Zn1XaTgW5GUQOctjuDAYjyWe3go5qtIHlTqG2vERb5LQ5gOxJ5uqEeBBj5BV+JoB9Vsi4pW83COfRa/bH6c2cGux7QREUgw2sZvP+toWjxOH7TCVGk3LJJ/eYSCfQNVS+gahplhh5pGD+8xzgQf76dFf4WsKmHfGpY4z8UkHMCPEFJ5pn4jMV8OX0gT7r2ZHcrOETCwlU5czSOIEgnwtZdU3Ya2ps8teM0tgRq1wmD4Tqub7z4R1Ku6fiAJ8Rwu+YnzWXmntq4b5iK1kUnEHkJvEzeI58lDCfm4I6leZWetDu3s1eTsuhJ07QCegqvAC0xVVulguwwGHpkXbSaTH7TuJ3zcValQEQlQgsUISKypVWbzUM+Eqt/dzf+pB+4qyQRNig4K5uWpxWLuE3sHE8Lg4W1Vxsx4NOHtJLmVARPxNcWtI7o+pVVt+pWZiKtN1JhyVHN5ZokwAPiBB0gqfsCvmptLuGO0aRHFOuoNtV1lUr02A6aOV1MSKjQXfE4ZmkB0TxATfprMKfWw+epTblJcS8gc5kQOKAFD2GwMlwfrVb7pBMgOJka3Fri6tnuDqlNzgGA5hYANbyi0ZRrcaKRAqU2EuF4mCWgzrMAmByI0OgUPA1HsANE54NLKKne5sZiJi4ElWuJrkEnKxwJsZeQI4Q7Xy4oVdgcQynV4y4hsEyHRIJ+EX1Os2UVLttOi12HZSqsDWupcbB8LjElpAiQcxnwjVc63wbQw9enRo5nHss1R5u0uLiBlMDoZA0suo1GMqtaTaPddOU3A08bKmr7DnE0n1GUqtNstymmQWEkuFQOkmRDRFhJmyz8nH1Ro4eXork+Ie+k7ia5t44gReJi/OIKvtgVjVdDQXHoASbeC2G9+GOJwmIw9epQdXdUBwrGVOydxQKBqBxu6Q6YsQ3uWa9mn5Pg31nVcXQqPztw7cmcFlaHl1NuYAOkN1/c71w/wAGsmn/AGZcfHdfnAVGxmYRIETEeZ5eavn1aj3gZeYktt7tgSb2u60fNJWojEU6IY/tGBzajnvmKjTJGkAmSCOQga2XhvBjWYHB1azYDmtMOmS55hrBm1cS5w+ZWjDDp8MvJydXlyf2hbS7fHVGznZTcaLNABB4mgzAGYwfsyuj4zhpuj9kDyDfxIXD3Vnl4LyS51TiMZjxESQHG4kldtxcBrgTePlI/ALZwsPP6Iy7qHPKxx/0wPqmUWRVf9mm0dIbn/FJs+o4kHL8BA8oH4qRUbla93USPCHn8F29uPmOZbAq9piKTdI7Z893aQB9VdbYpOweJD2t/N1dejX6OHgdfMql3Ow016b/ANmGeVQZr/xGPNdK2ps9uIoOY7SLdQdJCrctVbW2A3PqZDiKTRJbWIaDYQ4lzSe4NIKz3tM2Q5rada5iRUIFhmjKR3SIurHY2LFDGQ88VTMx0mONlgY7xPom72bcxFRldjMoY0GBEy34tfNV5dXHS2G5nuOaSgWv9QkmICe1hdwg3sB5mFhbX0ns2rnoUnftU2H1aFIUfZmH7KhSp/sU2N9GgKQgRCEqCelQhWVCEIQc09pOxJrNrNyjPZxI0cLC6q9g03SWnI8tFwIJbIPMD3rHujyW535tSa6Tq4dxmCAe6xXOthYUU8RVgmHMzm8uBbyPLRxiJsumKtWuBonhGX4yXEESSKbveNyIkdLKZiMOW3cZc05MhMPBPMC3T5heFMkOBBtmdyj4CRY8+SdiamZ13EuMCTY9MpJIHS/4KyEXaFDjDKeZxdGUNMmZNjovCrSe0Op1HFomS0NHO2YZoNi28QLJ2NGVzg+XMEfGGtzDRxEcep0nxUam9hcZIBkkHiAMg6loJGg5D5qEu37tYkVsHQqETnpM1HMNCsKogX1kGeokWueayXs3xw/JnUe0Y91J5ILHS3I+S0xq2+YQY0WtffXpFz1VEqnau7OFxUOqURmDxUzNe9hD2mWnhIk3NypA2BhJz9gzMHuqtJY21RxJLtIJEuub311U5jjIgaG1u65TnOn0BP3/AHppOzGUwBFgAQIAEDo0W0WD9q2JNV9LCMc2MvbPBc0SGh2VtyL2cZ8Fva9RrWy4wBck6Aal3lC+fdv7VOKxVWuScrjaLOFPRoHfASCFg8LSdVa4EEtrMApnhcZd8IaTmAIHTVdj2xYPP92XGtk06X5Th4ME16Jl3MGo2QT15CwXbtqiW1R4/SVp4WbnRNkCYgnSPXX6L029WyYeqR8LHeUM/qvPYsNaJ5WJ9P6+qjbdfnweIeTYUq0R+8HAH0XXX5OM8MPuwwh1USRlFIjxHZP/AOS6pSE5lzjYMN7awHCD5hjPnZdGwQt1/wClTPzVp6ci38wBZV7Vo0qTI53/AKqkxDnubUBdwvB5XbImxHf1XU989nh7XwJ5+kxHquJ7X2q6DSa3LBIe43cfDoPmuPJdOvHLeylYJ0Xvhf0jI1zt/mC8QB/einbLpZ6tJvN1WmI8XgT81man0i3QeAQU4iE0oEQhCCwQlQpVIhKhBRb4/q38Q+hXO9hfrzPsf8ShC64+FasT/h/YH8rkzaX6N3gPuQhWVipxvvu+yEmK1H2W/wAoSoULNd7Kv1ip/ks+q6diNB9r7ihCpUwUvfd5/epDNfL7kIQU+836piP8h/0Xz3itW/YP/JCEErB/p8F/mUv94Lt21Pcref8AKUIWjhZ+dBp6VP7+FQsf/wDW1f8AJd/I5CFoZ/bNYf8Axfst/wBtq6Bhvc/hQhcsvNXiLt73fT6r5y3h/Wqv2z9AhC4cv6x34f2qA7l4K13d/XsL/wDoof7jUIWdpfRrk0oQgRCEIP/Z" />
```

Comment:
why is this a huge data URI in the file?  We have a webpack loader that should do this for you-- you can `import joyHeadsot from './joy.jpg';` and then `src={joyHeadshot}`